packages/docusaurus/src/client/baseUrlIssueBanner/styles.module.css

```diff
@@ -0,0 +1,3 @@
+.baseUrlIssueBanner {
```

Comment:
Hmm, the header copyright check in CSS files apparently does not work?

Reply:
does not look like, but just added it